fighthealthinsurance/models.py

```
@@ -340,12 +342,34 @@ class FaxesToSend(ExportModelOperationsMixin("FaxesToSend"), models.Model):  # t
         "Appeal", on_delete=models.SET_NULL, null=True, blank=True
     )
 
+    def _get_contents(self):
+        if self.combined_document:
+            return self.combined_document.read()
+        elif self.combined_document_enc:
+            cryptographer = Cryptographer(settings.COMBINED_STORAGE)
+            try:
+                return cryptographer.decrypt(self.combined_document_enc.read())
+            except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)